src/main/java/org/jboss/xavier/integrations/rbac/RBACService.java

```diff
@@ -0,0 +1,262 @@
+package org.jboss.xavier.integrations.rbac;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RBACService {
```

Comment:
This seems to be more a `RBACUtils` with static methods
```suggestion
public class RBACUtils {
```

Reply:
renamed